gradle.properties

```diff
@@ -1,4 +1,4 @@
-version=29.26.3
+version=29.26.4
```

Comment:
You introduce new APIs in AvroUtil, it should bump minor version rather than patch version.

Reply:
adopted